guides/common/modules/con_converting-a-host-to-rhel.adoc

```diff
@@ -62,6 +62,23 @@ endif::[]
 ifndef::managing-hosts[]
 For more information, see {ManagingHostsDocURL}Registering_Hosts_by_Using_Global_Registration_managing-hosts[Registering Hosts by Using Global Registration] in _{ManagingHostsDocTitle}_.
 endif::[]
+. Run the pre-conversion analysis on the host group to verify if your hosts are ready for the conversion.
+Execute a remote job with the following settings:
+* **Job category**: `Convert 2 RHEL`
+* **Job template**: `Convert2RHEL analyze`
+
++
+ifdef::managing-hosts[]
+For more information, see xref:executing-a-remote-job_{context}[].
+endif::[]
+ifndef::managing-hosts[]
+For more information, see {ManagingHostsDocURL}executing-a-remote-job_managing-hosts[Executing a Remote Job] in _{ManagingHostsDocTitle}_.
+endif::[]
+
++
+Review pre-conversion analysis reports and resolve any issues that prevent the conversion.
+Repeat this step until you resolve all issues.
```

Comment:
I would add `resolve all issues that are blocking the conversion`

Some of the issues are not always problems users have to solve.